src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,71 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
+    Sign sign;
+
+    public int calculate(String inputString){
+        for(String splitData : inputString.split(" ")){
+                A(splitData);
+        }
+        return numbers.pop();
+    }
+
+    private void A(String splitData) {
+        switch (sign.getValue(splitData)){
+            case ADDITION:
+                numbers.push(addition());
+                break;
+            case SUBTRACTION:
+                numbers.push(subtraction());
+                break;
+            case MULTIPLICATION:
+                numbers.push(multiplication());
+                break;
+            case DIVISION:
+                numbers.push(division());
+                break;
+            case NUMBER:
```

Comment:
Learn thing what you are using. Switch operator have keyword for some default case. So you don't need this NUMBER